src/components/simple-billboard/index.js

```diff
@@ -0,0 +1,101 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+
+import { Icon } from 'nr1';
+
+/**
+ * @param {Object} metric - metric name, value, previousCalue
+ * @param {string} prefix - prefix for metric.value (i.e. '$')
+ * @param {string} suffix - suffix for metric.value (i.e. users)
+ * @param {string} className -  SCSS class name - gets added to existing JSX classes
+ * @param {Object} style - SCSS style - gets added to JSX - overwrites existing/duplicate CSS properties
+ * @return {JSX Object} - RENDERING name, value, up/down trend when previousValue present
+ */
+const SimpleBillboard = ({ metric, prefix, suffix, className, style }) => {
+  const metricValue = useMemo(
+    () => (isNaN(metric.value) ? '-' : Number(metric.value)),
+    [metric.value]
+  );
+  const metricPreviousValue = useMemo(
+    () => (isNaN(metric.previousValue) ? '-' : Number(metric.previousValue)),
+    [metric.previousValue]
+  );
+  const difference = useMemo(
+    () => metricValue - metricPreviousValue,
+    [metricValue, metricPreviousValue]
+  );
```

Comment:
The reason you have to check `isNaN(difference)` in `changeStatus` is because we're not checking in here! Let's update this to...
```javascript
  const difference = useMemo(
    () => typeof metricValue === 'number' && typeof metricPreviousValue === 'number' 
      ? metricValue - metricPreviousValue
      : 0,
    [metricValue, metricPreviousValue]
  );
```